README.md

```diff
@@ -48,6 +48,8 @@ To use, go to address generated by flask app in a browser.
 
 ### Running tests for flask backend
 
+Runs on python 3.4,3.5
+
 Tested on python 3.6
```

Comment:
Maybe instead of the addition above, something like,

"Tested on Python 3.4, 3.5 and 3.6"